### PR TITLE
[HOLD] MS BUILD Webinar and Topic pages

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

File: _config/protected.yml [hunk 1]
```diff
@@ -10,3 +10,6 @@
 - /video/youtube/i1aiUaa8RZ8/
 - /video/youtube/LbzkY9nrgdo/
 - /video/youtube/Q37niG_hCcA/
+- /video/youtube/pIURFD15WJ0/
+- https://plus.google.com/events/cmoaedv5ib92rlhh9bnjrkd1hhg?authkey=CNnNnODpxtTzbQ
```
Comment (paulrobinson) [inline _config/protected.yml]: You can't protect pages that are off-site. What do you want to achieve here?

File: _config/protected.yml [hunk 2]
```diff
@@ -10,3 +10,6 @@
...
+- /webinars/net-on-rhel-sneak-peek
```
Comment (paulrobinson) [inline _config/protected.yml]: I believe this is the landing page. We shouldn't protect this page, as we want to use this to encourage users to register.

File: _config/protected.yml [hunk 3]
```diff
@@ -10,3 +10,6 @@
 - /video/youtube/i1aiUaa8RZ8/
 - /video/youtube/LbzkY9nrgdo/
 - /video/youtube/Q37niG_hCcA/
+- /video/youtube/pIURFD15WJ0/
```
Comment (paulrobinson) [inline _config/protected.yml]: I've added this video to a playlist watched by the site. It should be available in this PR build after a rebuild. It'll be on production in < 6 hours.

**Comment by paulrobinson** [comment]
You also need to protect the registration-confirmation page too I think.

File: webinars/net-on-rhel-sneak-peek.html.slim [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      p
+        | You may have heard Microsoft hearts Linux.  And soon you will be able to use .NET on Red Hat Enterprise Linux. While the pieces are still being finalized we are excited to share a behind-the-scenes sneak peek at how this will work for you.
+      p
+        | In this 2-hour webinar and demo, see:
+        ul
+          li How .NET on Red Hat Enterprise Linux is coming together
+          li What you can expect from the development experience
+          li .NET on Red Hat Enterprise Linux in action
+        strong Register now for this unique sneak peek.
+      a.button.long(href="#") Register Now
```
Comment (paulrobinson) [inline webinars/net-on-rhel-sneak-peek.html.slim]: This button doesn't do anything. It should go to the 'registration-confirmation' page.

File: _config/protected.yml (hunk 2 above)
Comment (paulrobinson) [inline _config/protected.yml]: Also, I noticed the reg-walling isn't working for http://developers-pr.stage.redhat.com/pr/956/build/655/webinars/net-on-rhel-sneak-peek/. I know I said above that we don't want that. But given the current code I'd have expected to be asked to login.

Comment (paulrobinson) [inline _config/protected.yml]: FYI: the reg-walling works for other pages in this PR: http://developers-pr.stage.redhat.com/pr/956/build/655//video/youtube/I-Po2phF-_M

**Comment by paulrobinson** [comment]
FYI, here's the video to link to: http://developers.redhat.com/video/youtube/pIURFD15WJ0/

**Comment by LightGuard** [comment]
Is this still on hold?

**Comment by adelasofia** [comment]
@LightGuard Yes, I'm about to push more changes. I think @paulrobinson will merge this when he gets back.

**Comment by adelasofia** [comment]
@paulrobinson Everything approved by Emily :+1:

**Pull request merged**